pipelines/A17/A17.R

```diff
@@ -0,0 +1,93 @@
+# -------------------------------------------------------------------------
+# Steps: Get the data
+# input: google sheet
+# output: ds$data, ds$dir
+# -------------------------------------------------------------------------
+
+ds <- statbotData::create_dataset("A17")
+ds <- statbotData::download_data(ds)
+
+# -------------------------------------------------------------------------
+# Step: Clean the data
+#   input: ds$data
+#.  output: ds$cleaned_data: data has been cleaned:
+#.  - factor `ergebnis` spilt into wide format
+#.  - `datum_und_vorlage` split into `jahr` and `vorlage_title`
+# -------------------------------------------------------------------------
+
+ds$data %>%
+  janitor::clean_names(
+  ) %>%
+  dplyr::mutate(
+    "jahr" = stringr::str_extract(datum_und_vorlage, '^\\d{4}')
+  ) %>%
+  dplyr::mutate(
+    "vorlage_title" = stringr::str_replace(datum_und_vorlage, '^\\d{4}-\\d{2}-\\d{2} ', "")
+  ) %>% dplyr::select(
+    -c(datum_und_vorlage)
+  ) %>%
+  dplyr::rename(
+    "anzahl" = volksabstimmungen_ergebnisse_ebene_kanton_seit_1866
+  ) %>%
+  tidyr::pivot_wider(
+    names_from = c("ergebnis"),
+    values_from = anzahl
+  ) %>%
+  janitor::clean_names(
+  ) %>%
+  dplyr::rename(
+    "anzahl_stimmberechtigte" = stimmberechtigte,
+    "anzahl_abgegebene_stimmen" = abgegebene_stimmen,
+    "beteiligung_in_prozent" = beteiligung_in_percent,
+    "anzahl_gueltige_stimmen" = gultige_stimmzettel,
```

Comment:
Detail: Not sure about renaming stimmzettel -> stimmen (as far as I understand, it is the actual "Zettel" that's invalid for technical reasons, not the person't vote). I'm worried this might be confusing.

Reply:
Done.